src/settings.ts

```diff
@@ -80,6 +83,17 @@ export class ThingsLogbookSettingsTab extends PluginSettingTab {
       });
   }
 
+  addDoesSyncToeBody() {
+    new Setting(this.containerEl)
+      .setName("Sync note's body")
```

Comment:
👀  Do you think this is more clear?

```suggestion
      .setName("Include 'Notes' field with tasks")
```

Reply:
Changed it a bit: 

```js 
      .setName("Include notes")
      .setDesc('Includes MD notes of a task into the synced Obsidian document')
```

So the name is short